english/invite-friends/smores/with_friends.md

```diff
@@ -1,4 +1,13 @@
 You ask your best friends to bring some chocolate and graham crackers.
 You pick marshmallows from the wall, toasting them with your lighter.
 You place the hot marshmallow and a piece of chocolate on the graham
-cracker and enjoy your new snack with your friends.
\ No newline at end of file
+cracker and enjoy your new snack with your friends.
+
+Now what?
+
+Do you want to:
+
+- [Sit back and relax](smores/after_smores/indigestion.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.